core/domain/html_cleaner.py

```diff
@@ -458,13 +460,127 @@ def convert_to_text_angular(html_data):
     return unicode(soup).replace('<br/>', '<br>')
 
 
-def convert_tag_contents_to_text_angular(html_data):
```

Comment:
Minor naming nit: I prefer we keep the editor names as one word, ie `convert_to_ckeditor` and `convert_to_textangular`

Reply:
Done!